fastembed/late_interaction/colbert.py

```diff
@@ -70,9 +82,12 @@ def _preprocess_onnx_input(
         self, onnx_input: Dict[str, np.ndarray], is_doc: bool = True
     ) -> Dict[str, np.ndarray]:
         if is_doc:
-            onnx_input["input_ids"][:, 1] = self.DOCUMENT_MARKER_TOKEN_ID
+            onnx_input["input_ids"][:, 1] = self.document_marker_token_id
         else:
-            onnx_input["input_ids"][:, 1] = self.QUERY_MARKER_TOKEN_ID
+            onnx_input["input_ids"][:, 1] = self.query_marker_token_id
+
+        if self.tokenizer_class == "XLMRobertaTokenizer":
```

Comment:
It is specific to jina models, if we have another model which uses `XLMRobertaTokenizer` and doesn't need this modification, then it will be broken.
We should create a separate class in the same way as it is done with dense text models